css/styles.css

```diff
@@ -0,0 +1,323 @@
+    margin-bottom:0px;
+ 
+}
+
+h1{
+
+    color:yellow;
+    font-family:georgia;
+    font-size:18px;
+    padding-left:10px;
+    margin-top:5px;
+    margin-bottom:5px;
+ 
+}
+
+p{
+
+    kerning:1px;
+    font-family:helvetica;
+    font-size:12px;
+}
+
+.aside-h2{
+
+    color:black;
+    font-kerning:10px;
+
+}
+
+
+nav{
+
+    width:100%;
+    height:40%;
+    font-size:12px;
+    background-color:black;
+    float:right;
+    overflow:auto;
+    border-bottom:3px solid yellow;
+    text-align:right;
+    display:inline;
+    padding-right:1px;
+
+}
+
+.nav-upper{
+
+    text-transform:uppercase;
+    font-family:helvetica;
+    font-weight:bold;
+    color:white;
+
+}
+
+.nav-lower {
+
+    /*text-transform:uppercase;*/
+    font-family:helvetica;
+    /*font-weight:bold;*/
+    color:white;
+
+}
+
+li{
+
+    display:inline;
+    margin-right:20px;
+}
+
+.sub-head{
+
+
+    width:100%;
+    height:10%;
+    background-color:#ccc;
+    overflow:auto;
+
+}
+
+.sub-head-text{
+
+    text-align:center;
+    padding-top:1px;
+    padding-bottom:1px;
+    margin-top: 15px;
+    margin-bottom:15px;
+    font-family:georgia;
+    font-style:italic;
+    font-weight:bold;
+
+p{
+
+    font-family:helvetica;
+    font-size: 14px;
+}
+
+}
+
+article{
+
+    width:100%;
+    height: 300px;
+    color:white;
+    font-family:helvetica;
+    font-size:36px;
+    background-image: url(../images/coworking.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+
+}
+
+
+aside{
+
+    width:175px;
+    height:270px;
+    color:black;
+    font-family:helvetica;
+    font-size:12px;
+    background-color:#ccc;
+    overflow:auto;
+    padding-right:10px;
+    padding-left:10px;
+    float:right;
+    border:1px black;
+    margin-bottom:20px;
+    margin-top:20px;
+    margin-right:20px;
+    opacity: 0.75;
+    filter: alpha(opacity=75); /* For IE8 and earlier */
+
+}
+
+.yellow-button{
+
+    width:155px;
+    height:30px;
+    overflow:auto;
+    font-size:12px;
+    font-weight:bold;
+    font-family:helvetica;
+    text-align:center;
+    padding-top:8px;
+    background-color:yellow;
+    opacity:100%;
+    filter: alpha(opacity=100); /* For IE8 and earlier */
+    text-transform:uppercase;
+    border-radius:5%;
+
+}
+
+.gray-button{
+
+    width:145px;
+    height:30px;
+    overflow:auto;
+    font-size:12px;
+    font-weight:bold;
+    font-family:helvetica;
+    text-align:center;
+    padding-top:8px;
+    background-color:#ccc;
+    text-transform:uppercase;
+    border-radius:5%;
+
+}
+
+.bottom-h1{
+
+    color:black;
+    font-family:georgia;
+    font-size:18px;
+    font-weight:bold;
+    padding-left:10px;
+    margin-top:5px;
+    margin-bottom:5px;
+    font-family:helvetica;
+
+
+}
+
+.content {
+  width: 100%;
+  overflow: auto;
+  clear: both;
+}   
+
+.section {
+  height: 200px;
+  background-color: white;
+  border-left:0.5px #ddd solid;
+  float: left;
+  width: 30%;
```

Comment:
Remove the margins since your blocks are almost equal percentages of its container. Applying margins offsets their position. Then adjust the width to compensate for the missing margin 🔨